test/scripts/tags/asset_link.js

```diff
@@ -49,11 +49,11 @@ describe('asset_link', () => {
   });
 
   it('should escape tag in title', () => {
-    assetLink('bar true "Hello" <world>').should.eql('<a href="/foo/bar" title="&quot;Hello&quot; &lt;world&gt;">&quot;Hello&quot; &lt;world&gt;</a>');
+    assetLink('bar "Hello" <world> true').should.eql('<a href="/foo/bar" title="&quot;Hello&quot; &lt;world&gt;">&quot;Hello&quot; &lt;world&gt;</a>');
```

Comment:
the purpose is to test default option, so shouldn't need to specify `true`.

Reply:
there are 3 test cases
- option omitted(default): should escape tag in title by default
- true: should escape tag in title
- false: should not escape tag in title

Reply:
sorry, missed that.